src/app/+browse-by/+browse-by-title-page/browse-by-title-page.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <div class="browse-by-title w-100 row">
     <ds-browse-by class="col-xs-12 w-100"
-      title="{{'browse.title' | translate:{collection: '', field: 'Title', value: ''} }}"
+      title="{{'Browsing {{ collection }} by {{ field }} {{ value }}' | translate:{collection: '', field: 'Title', value: ''} }}"
```

Comment:
I get the exception below on this label. That likely means all the other identical labels in other components will cause the same exception:
```
Parser Error: Unexpected token Lexer Error: Unterminated quote at column 24 in expression ['Browsing {{ collection ] at column 25 in [{{'Browsing {{ collection }} by {{ field }} {{ value }}' | translate:{collection: '', field: 'Title', value: ''} }}] in ng:///BrowseByModule/BrowseByTitlePageComponent.html@3:6
```

Reply:
Whenever there's the two curly bracket sequence, I've been breaking it into separate strings cfr https://stackoverflow.com/questions/16868024/how-do-i-escape-curly-braces-for-display-on-page-when-using-angularjs

For example, footer component or pagination component htmls.